vault-harvester/scripts/deploy.ts

```diff
@@ -63,12 +63,24 @@ const deployContract = async (
     contract: "",
     impl: "",
   };
+
   const deployTx = await contractFactory.deploy(...constructorArgs);
   await deployTx.deployed();
   console.log(`${name}: ${deployTx.address}`);
   const contract = await ethers.getContractAt(name, deployTx.address);
   ret.contract = contract.address;
 
+  if (network.name !== "hardhat" && network.name !== "localhost") {
+    console.log("Waiting to verify...");
+    await deployTx.deployTransaction.wait(6);
+    console.log(`Verifying ${config.harvester.contractName}...`);
+    await run("verify:verify", {
+      address: deployTx.address,
+      contract: `contracts/test/${config.harvester.contractName}.sol:${config.harvester.contractName}`,
+      constructorArguments: constructorArgs,
+    });
+  }
+
```

Comment:
Can we move this to `deployHarvester` because `deployContract` is a generic function which only works with its parameters as input.